Trading/Mode/market_maker_trading_mode.py

```diff
@@ -0,0 +1,206 @@
+
+            for order_quantity, order_price in self.check_and_adapt_order_details_if_necessary(quantity,
+                                                                                               limit_price,
+                                                                                               symbol_market):
+                current_order = trader.create_order_instance(order_type=TraderOrderType.SELL_LIMIT,
+                                                             symbol=symbol,
+                                                             current_price=price,
+                                                             quantity=order_quantity,
+                                                             price=order_price)
+                updated_limit = await trader.create_order(current_order, portfolio)
+                created_orders.append(updated_limit)
+
+                await trader.create_order(current_order, portfolio)
+
+            # Create LONG order
+            quantity = self._get_quantity_from_risk(trader, market_quantity)
+            quantity = self.add_dusts_to_quantity_if_necessary(quantity, price,
+                                                               symbol_market, current_symbol_holding)
+
+            limit_price = best_bid_price - (best_bid_price * delta_bid * quantity)
+
+            for order_quantity, order_price in self.check_and_adapt_order_details_if_necessary(quantity,
+                                                                                               limit_price,
+                                                                                               symbol_market):
+                current_order = trader.create_order_instance(order_type=TraderOrderType.BUY_LIMIT,
+                                                             symbol=symbol,
+                                                             current_price=price,
+                                                             quantity=order_quantity,
+                                                             price=order_price)
+                await trader.create_order(current_order, portfolio)
+                created_orders.append(current_order)
+
+            return created_orders
+
+        except InsufficientFunds as e:
+            raise e
+
+        except Exception as e:
+            get_logger(self.__class__.__name__).error(f"Failed to create order : {e}. "
+                                                      f"Order: "
+                                                      f"{current_order.get_string_info() if current_order else None}")
+            get_logger(self.__class__.__name__).exception(e)
+            return None
+
+
+class MarketMakerTradingModeDecider(AbstractTradingModeDecider):
+    def __init__(self, trading_mode, symbol_evaluator, exchange):
+        super().__init__(trading_mode, symbol_evaluator, exchange)
```

Comment:
Remove this line ? Only calling superClass

Reply:
Yes